src/main/proto/netflix/titus/titus_job_api.proto

```diff
@@ -187,16 +187,28 @@ message JobDisruptionBudget {
         uint32 limit = 1;
     }
 
-    /// Maximum number of times a task can be evicted.
-    message EvictionLimit {
+    /// Maximum number of times a task can be relocated.
+    message RelocationLimit {
         uint32 limit = 1;
     }
 
     oneof Policy {
         AvailabilityPercentageLimit availabilityPercentageLimit = 1;
         AvailableTasksLimit availableTasksLimit = 2;
-        EvictionLimit relocationLimit = 3;
+        RelocationLimit relocationLimit = 3;
     }
+
+    /// Defines the relocation delay. A user is notified about the
+    //  relocation plan, and may choose to handle the relocation itself.
+    //  If the contain is still running after the delay passes, it is
+    //  moved by the task relocation process.
+    uint64 relocationDelayMs = 4;
```

Comment:
Remove this and just add a `Self Managed` policy or mechanism?

Reply:
Done